### PR TITLE
Add org.srb2.SRB2Kart-Saturn

### PR DESCRIPTION
**Pull request by riomccloud**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/Indev450/SRB2Kart-Saturn
image
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
No changes were made

SRB2Kart Saturn is a net-compatible custom build of the game (like SRB2Kart Galaxy) that adds a range of features, from code optimizations to game improvements.

### CODE REVIEW

File: org.srb2.SRB2Kart-Saturn.yaml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  - name: sdl-mixer
+    buildsystem: autotools
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/*.a
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL_mixer/archive/refs/tags/release-2.6.3.tar.gz
```
Comment (hfiguiere) [inline org.srb2.SRB2Kart-Saturn.yaml:71]: this is in the run time:

```
[📦 org.freedesktop.Sdk ~]$ pkg-config --modversion SDL2_mixer
2.6.3
```

File: org.srb2.SRB2Kart-Saturn.yaml [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      - make -C src -j $FLATPAK_BUILDER_N_JOBS NOUPX=1 SDL=1 NOLIBBACKTRACE=1 HAVE_DISCORDRPC=1 $ARCH_MAKE_ARGS
+      - install -D -m 755 bin/Linux*/Release/lsdl2srb2kart $FLATPAK_DEST/bin/srb2kart
+      - install -D -m 644 srb2.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+      - install -D -m 644 src/sdl/srb2icon.png $FLATPAK_DEST/share/icons/hicolor/64x64/apps/$FLATPAK_ID.png
+    sources:
+      - type: git
+        url: https://github.com/Indev450/SRB2Kart-Saturn.git
+        commit: 040faa4c1c59232e29b462116e951da9468ed563
+
+  - name: other-files
```
Comment (hfiguiere) [inline org.srb2.SRB2Kart-Saturn.yaml:152]: this should be in the same module as the main program

**Comment by hfiguiere** [comment]
bot, build org.srb2.SRB2Kart-Saturn

File: org.srb2.SRB2Kart-Saturn.yaml (hunk 1 above)
Comment (riomccloud) [reply org.srb2.SRB2Kart-Saturn.yaml]: Removed 👍🏻

File: org.srb2.SRB2Kart-Saturn.yaml (hunk 2 above)
Comment (riomccloud) [reply org.srb2.SRB2Kart-Saturn.yaml]: Merged 👍🏻

**Comment by riomccloud** [comment]
I made the changes requested and fixed the xml. Tested here, shouldn't fail this time

File: org.srb2.SRB2Kart-Saturn.desktop
Comment (bbhtt) [inline org.srb2.SRB2Kart-Saturn.desktop:1]: Can you submit the desktop file and appdata to upstream?

**Comment by bbhtt** [comment]
bot, build org.srb2.SRB2Kart-Saturn

**Review by riomccloud — COMMENTED** [review]
Sorry, I didn't understand. The appdata and the desktop file are already here.

File: org.srb2.SRB2Kart-Saturn.desktop
Comment (bbhtt) [reply org.srb2.SRB2Kart-Saturn.desktop:1]: Here https://github.com/Indev450/SRB2Kart-Saturn

**Comment by barthalion** [comment]
If they are already in upstream. any reason you're maintaining a copy here?

**Comment by riomccloud** [comment]
Sorry for the delay.
> If they are already in upstream. any reason you're maintaining a copy here?
I thought they weren't necessary, since they're just used by the flatpak. However, I sent it to mainstream, just waiting the main developer. :)

**Comment by riomccloud** [comment]
Hello, is there something I still need to fix?

**Comment by bbhtt** [comment]
> Hello, is there something I still need to fix?

Since it is upstream can you bump the tag/commit to pull desktop, appdata etc. from there and remove the ones here?

**Comment by riomccloud** [comment]
Done.

**Comment by bbhtt** [comment]
bot, build org.srb2.SRB2Kart-Saturn

**Pull request closed without merging**